.github/workflows/lint.yml

```diff
@@ -56,6 +56,33 @@ jobs:
         export ADDITIONAL_LINTRUNNER_ARGS="--skip CLANGTIDY,CLANGFORMAT"
         .github/scripts/lintrunner.sh
 
+  # NB: This doesn't work with forked PR https://github.com/parkerbxyz/suggest-changes/issues/33 :(
+  lintrunner-autoformat:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: "linux.2xlarge"
```

Comment:
We should use `lf.linux.2xlarge` here (I'm seeing queue for `linux.2xlarge` so I wonder if we have already scaled it down)